react/components/AccordionFilterItem.js

```diff
@@ -49,11 +48,11 @@ const AccordionFilterItem = ({
           return (
             <Link
               key={opt.Name}
-              className="vtex-accordion-filter__item-opt pv3 ph7 pointer bb b--light-gray link fw3 db dark-gray"
+              className="vtex-accordion-filter__item-opt pv3 ph7 pointer bb b--muted-4 link fw3 db c-muted-1"
               page={pagesArgs.page}
               params={pagesArgs.params}
               query={pagesArgs.queryString}
-              onClick={onItemSelected}
+              onClick={onItemSelected}link
```

Comment:
Lint not working or is this misplaced?

Reply:
Not working here, still don't know why. Trying to fix it